.github/workflows/ecs.yml

```
@@ -1,24 +1,24 @@
+      - "CHANGELOG.md"
+      - "docs/**"
+      - "README.md"
 
-  push:
-    paths-ignore:
-      - 'docs/**'
-      - 'README.md'
-      - 'CHANGELOG.md'
-      - '.gitignore'
-      - '.gitattributes'
+  push: *ignore-paths
```

actionlint (1.7.7)

"push" section is alias node but mapping node is expected

(syntax-check)